CMakeLists.txt

```diff
@@ -186,15 +188,36 @@ endif()
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
 # build the parson library for json parsing
-add_library(parson
-    ./deps/parson/parson.c
-    ./deps/parson/parson.h
-)
-if (MSVC)
-    set_source_files_properties(../deps/parson/parson.c PROPERTIES COMPILE_FLAGS "/wd4244 /wd4232")
+if(NOT ${use_installed_dependencies})
+  add_library(parson
+      ./deps/parson/parson.c
```

Comment:
Isn't a cmake recommendation not to use relative paths (. and ..)? They should use CMAKE_CURRENT_LIST_DIR

Here and in other places below.